ehitchhiking-ui/src/app/shared/components/no-data/no-data.component.sass

```diff
@@ -0,0 +1,34 @@
+@import "../../../../styles/fonts-constants"
+
+.no-data-block-lg
+  margin-top: 30%
```

Comment:
Could you please clarify why this offset is required?

Reply:
fixed